### ISSUE TITLE
Adapt the specification to Git's official terminology and recommendations

### ISSUE THREAD

**Issue by tristan-cottam** [conventional-commits/conventionalcommits.org#518]
I noticed that [Git's documentation about `git commit`](https://git-scm.com/docs/git-commit) itself recommends limiting the commit "description" length to "less than 50 character[s]" (which I dare rather interpret as "50 characters or less" — it doesn't mention anything about the 72-character "body" rule), as well as using the term "title" instead of "description" and "description" instead of "body".

In a nutshell, I think the specification should be more strict and explicit, basing itself on Git's official documentation, dare I say even being more opinionated at times (while not being part of the official documentation, a soft limit on the commit description's line length should be specified), in order to achieve further standardization.
